ontology/yaml/resources/HVAC/entity_types/MAU.yaml

```diff
@@ -660,6 +660,21 @@ MAU_SFVSC_CHWSC_SSPC_FDPSM2X:
   - FDPSM2X
   opt_uses:
   - supply_air_isolation_damper_status 
+
+MAU_CHWSC_HWSC_SSPC_SFSS_SFVSC_FDPM2X_SFM_OA:
+  description: "Multi-zone MAU with chilled water and heating water."
+  is_canonical: true
+  implements:
+  - MAU
+  - CHWSC
+  - HWSC
+  - SSPC
+  - SFSS
+  - SFVSC
+  - FDPM2X
+  - SFM
+  - OA
```

Comment:
this is usually assigned to its own device (a weather station). is this really used by this device?

Reply:
bump on this.